cmd/build_test.go

```diff
@@ -1188,6 +1188,9 @@ p[k] contains v if {
 			expectedFiles: map[string]string{
 				"/optimized/test/p.rego": `package test.p
 
+
+# METADATA
+# entrypoint: true
```

Comment:
Should this have been omitted in the optimized module since the optimized package was an entrypoint?

Reply:
We are maintaining current behavior here in terms of not annotating a shifted optimized entrypoint. Can we handle that as part of a separate change? We can probably add a note in the docs for now. WDYT @johanfylling?

Reply:
Sure.

Would be good if we could inform the user about this, somehow. We don't have much in ways of logging when building a bundle, right? 
Maybe we could leave a comment in the original source file if we've dropped an entrypoint. E.g.:

```rego
# NOTE: Entrypoint foo/bar/baz has been dropped due to build optimization.
```

Reply:
> Would be good if we could inform the user about this, somehow. We don't have much in ways of logging when building a bundle, right?

We already do that if you run `opa build` with the `--debug` flag. So for the above policy we log a message like
```
optimizer: entrypoint: data.test.p: discard due to self-reference
```

WDYT?